### ISSUE TITLE
[Feature Request] Un-render received markdown emails

### ISSUE THREAD

**Issue by Nikita240** [adam-p/markdown-here#252]
When you copy a rendered markdown email, the plaintext in your clipboard does not contain the same handful formatting. It would be nice to be able to convert received markdown emails back to plaintext.

**Comment by adam-p** [comment]
An easy work around for this occurred to me -- use the fact that the "forward" compose box shows the previous content pretty directly. So it's just these steps:

1. Open the message.
2. Start a forward of it.
3. Markdown Toggle.
4. The forward content will be replaced with the original Markdown.

I tested in Gmail and Thunderbird. If the mail client quotes the forwarded content, then it won't work.

I added this to the [Tips and Tricks](https://github.com/adam-p/markdown-here/wiki/Tips-and-Tricks#post-send-md) wiki page.
